.github/workflows/node.js.yml

```diff
@@ -307,3 +307,54 @@ jobs:
         with:
           name: cypress-screenshots
           path: test/cypress/cypress/screenshots
+  
+  publish:
+    name: Publish packages
+    runs-on: ubuntu-latest
+    needs:
+    - integration-tests
+    - cypress-static-plugin-auth
+    - core-components-test
+    - cypress-generateApp-monorepo
+    - cypress-generateApp-standalone
+    - cypress-generateApp-standaloneCLI
+    if: ${{( github.head_ref == 'next' || github.head_ref == 'master') && github.actor!= ‘dependabot[bot]’}}
+    steps:
+      - name: 📚 Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: "0"
+        run: git fetch --depth=1 origin +refs/tags/*:refs/tags/*
+
+      - name: 🟢 Setup Node ${{ matrix.node_version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+
+      - name: ⏳ Install
+        run: |
+          yarn --frozen-lockfile
+          yarn lint
+          yarn bs
+
+      - name: 🗝 Configure git credentials
+        uses: OleksiyRudenko/gha-git-credentials@v2
+        with:
+          token: ${{ secrets.GH }}
+
+      - name: 🐒 Authenticate with NPM
+        run: |
+          yarn logout
+          echo "registry=http://registry.npmjs.org/" >> .npmrc
+          echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" >> .npmrc
+          npm whoami
+        env:
+          NPM_TOKEN: ${{ secrets.NPM }}
+
+      - name: 🚽 Publish package
+        run: yarn lerna publish --yes ${{LERNA_TAG}}
+        env:
+          GH_TOKEN: ${{ secrets.GH }}
```

Comment:
where is `secrets.GH` coming from? There is already `{{ secrets.GITHUB_TOKEN }}` globally available if thats of any use?

Reply:
changed to GITHUB_TOKEN